package/harvester-os/files/usr/sbin/cos-installer

```diff
@@ -0,0 +1,497 @@
+#!/bin/bash
```

Comment:
cOS recently introduced a [post-install stage](https://github.com/rancher-sandbox/cOS-toolkit/pull/487).
Let me see if I can use it. We just need to do more preparation and cleanup like mount and umount target OS.

Reply:
I don't use the after-install and before-install cOS stage, because the UX is not good. ([execution output is discarded and error status is not checked](https://github.com/rancher-sandbox/cOS-toolkit/blob/0d402c163aa2e1b6c36ef23ff928c1392579dba8/packages/installer/installer.sh#L387))
Instead I create a [new script](https://github.com/harvester/harvester-installer/pull/93/commits/84794e1d338982cd10b2723dfe95b6bd784ee6ad#diff-6a84a5d8c064b0be0d59d2d26e27e2549fcf932c02014eb3f954bec7e780d7f7) to wrap the cos-installer script.
The script contains 3 parts:
- pre: get ISO (only in PXE boot)
- cos install: run un-modified `cos-installer`
- post: pre-load harvester images and other customization.